tensorbay/client/gas.py

```diff
@@ -355,6 +355,26 @@ def list_dataset_names(self) -> PagingList[str]:
             lambda offset, limit: self._generate_dataset_names(None, offset, limit), 128
         )
 
+    def update_dataset(
+        self,
+        name: str,
+        *,
+        alias: str = ...,  # type: ignore[assignment]
+    ) -> None:
+        """Update a TensorBay Dataset.
+
+        Arguments:
+            name: Name of the dataset, unique for a user.
+            alias: New alias of the dataset.
+
+        """
+        dataset_id = self._get_dataset(name)["id"]
+        patch_data = {}
```

Comment:
what if `alias` is `...` and `patch_data` is `{}`

Reply:
The request will be sent only if patch_data is not empty.